src/main/java/com/ekart/thunderbolt/utils/CommonUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.ekart.thunderbolt.utils;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public class CommonUtils {
+
+    public static boolean isStringEmpty(boolean includeNullStringAlso, String... parameters){
+        for(String param : parameters){
+            if(param == null || param.length() == 0){
+                return false;
+            } else if(includeNullStringAlso && "null".equals(param)) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    public static Double roundOffDouble(Double rate){
+        return roundOffDouble(rate, null);
+    }
+
+    public static Double roundOffDouble(Double rate, String field){
+        if(rate != null){
+            return (Math.round(rate * 100.00) / 100.00);
+        } else {
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST);
```

Comment:
Utils class must not return ResponseStatusException, which is exclusive for the classes that deal with request/response. The correct way here would be create a new exception class that extends RuntimeException and  catch it in the class that calls the method, if needed.